years.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if (grep -r "Copyright[[:space:]]\+(c)[[:space:]]\+2009-.*rultor.com" --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . | grep -v 2009-`date +%Y`) then
```

Comment:
@mkordas It was concern of @original-brownbear, you can see comment [here ](https://github.com/yegor256/rultor/issues/981#issuecomment-189685079)